modules/identity/src/Volo.Abp.Identity.Domain/Volo/Abp/Identity/IdentityRole.cs

```diff
@@ -49,6 +49,11 @@ public class IdentityRole : AggregateRoot<Guid>, IMultiTenant
     /// </summary>
     public virtual bool IsPublic { get; set; }
 
+    /// <summary>
+    /// An entity version property that auto-increments when the entity changes.
```

Comment:
```suggestion
    /// A version value that is increased whenever the entity is changed.
```

Can you also change the others, and also the one in the `IHasEntityVersion` interface.